homeassistant/components/wemo/__init__.py

```diff
@@ -149,7 +150,16 @@ async def async_add_unique_device(
         if wemo.serialnumber in self._added_serial_numbers:
             return
 
-        coordinator = await async_register_device(hass, self._config_entry, wemo)
+        try:
+            coordinator = await async_register_device(hass, self._config_entry, wemo)
+        except pywemo.PyWeMoException as err:
+            if wemo.serialnumber not in self._failed_serial_numbers:
```

Comment:
We need to remove them from the set once they are active again right?

Reply:
Sounds good. Done in [a4aaed2](https://github.com/home-assistant/core/pull/91621/commits/a4aaed27b09e8f2be2b7e2e4c3a8e6a0cd1f959c)